internal/compiler/parser.rs

```diff
@@ -916,6 +956,13 @@ impl NodeOrToken {
             NodeOrToken::Token(t) => t.text_range(),
         }
     }
+
+    pub fn next_sibling_or_token(&self) -> Option<Self> {
```

Comment:
where is this used?

Reply:
That indeed seems unused now. I'll double check when I am done with the whole thing...

I only added a method here when I needed, but I also changed the user side a lot over time...

Reply:
Removed again... incl. the code it calls on `SyntaxToken` and `SyntaxNode`.